Jenkinsfile

```diff
@@ -74,9 +77,13 @@ def job = {
 
         // For a release build we remove the -SNAPSHOT from the version.
         config.ksql_db_version = config.ksql_db_version.tokenize("-")[0]
+
+        config.ksql_db_artifact_version = config.ksql_db_version
         config.docker_tag = config.ksql_db_version
     } else {
-        config.docker_tag = config.ksql_db_version.tokenize("-")[0] + '-' + env.BUILD_NUMBER
+        // For non-release builds, we append the build number to the maven artifacts and docker image tag
+        config.ksql_db_artifact_version = config.ksql_db_version + '-' + env.BUILD_NUMBER
```

Comment:
if you are going to append the build number then you need to remove the snapshot qualifier. i made a more detailed comment about this in the cloud PR since I reviewed that one first

Reply:
Yup, responded on the other PR. Updated here as well.